vars-sample.yml

```diff
@@ -14,6 +14,11 @@ remote_web_user_password: "EncryptedSecretString"
 # created.  The default is usually sufficient if you've already got keys generated in the default location.
 web_user_ssh_public_key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
 
+## -- PHP configuration -- ##
+php_fpm:
+  # PHP version to install. Defaults to PHP 7.3.
+  version: 7.2
```

Comment:
comment says defaults to `7.3`, but the actual version configured is `7.2` - typo?

Reply:
This is just an example, implying that you can use any PHP version there, but it will default to 7.3 if `php_fpm.version` not provided. Make the comment more descriptive ?

Reply:
ahh missed that.  It's okay as is.